openid-federation-1_0.xml

```diff
@@ -1378,77 +1378,43 @@
-	      This is done via request authentication methods.
-	      These methods achieve this by proving that
-	      the RP is in possession of a key that appears in its metadata.
-	      This parameter declares the request authentication methods that the OP supports.
-	      <vspace blankLine="1"/>
-
-	      The <spanx style="verb">request_authentication_methods_supported</spanx>
-	      value is a JSON object where the member names are names of endpoints
-	      where the request authentication occurs.
-	      This MAY be either at
-	      the OP's Authorization Endpoint or
-	      the OP's Pushed Authorization Request (PAR) endpoint.
-	      Supported endpoint identifiers are
-	      <spanx style="verb">authorization_endpoint</spanx> and
-	      <spanx style="verb">pushed_authorization_request_endpoint</spanx>.
-	      The values of the JSON object members for the endpoint names are
-	      JSON arrays containing the names of the request authentication methods
-	      used at those endpoints.
-	      <vspace blankLine="1"/>
-
+	      A Pushed Authorization Request (PAR) <xref target="RFC9126"/>
+	      can be used with Automatic Registration,
+	      as described in <xref target="authn-request"/>.
 	      Some OAuth 2.0 client authentication methods can be used at the
 	      <spanx style="verb">pushed_authorization_request_endpoint</spanx>
 	      to achieve request authentication.
 	      They are the subset of those listed in the
 	      IANA "OAuth Token Endpoint Authentication Methods" registry <xref target="IANA.OAuth.Parameters"/>
 	      that perform client authentication by proving possession of a private key.
-              These client authentication methods are:
+	      The <spanx style="verb">pushed_authentication_methods_supported</spanx>
+	      metadata value lists the client authentication methods
+	      supported by the OP that can be used in this manner.
+	      Its value is a JSON array of supported client authentication methods.
+	      Values that MAY be used are:
               <list style="hanging">
                 <t hangText="private_key_jwt">
                   <vspace/>
                   This client authentication method is described in Section 9 of
                   <xref target="OpenID.Core">OpenID Connect Core 1.0</xref>.
-                  Note that if <spanx style="verb">private_key_jwt</spanx> is
-                  used, the audience of the signed JWT MUST be either the URL of
-                  the Authorization Server's Authorization Endpoint or the
-                  Authorization Server's Entity Identifier.
-                </t>
-                <t hangText="tls_client_auth">
-                  <vspace/>
-                  Section 2.1 of
-                  <xref target="RFC8705"/>.
+                  When <spanx style="verb">private_key_jwt</spanx> is used,
+		  the audience of the signed JWT MUST be
+                  the Authorization Server's Entity Identifier.
```

Comment:
Does this restriction also apply to Federation deployments that do not use the `pushed_authentication_methods_supported` metadata parameter, but nonetheless still support PAR? (since the metadata parameter is marked as optional)